modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Bare Metal Operator]
+ifdef::cluster-caps[= Bare metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Bare Metal Operator is an optional cluster capability that can be disabled by cluster administrators if it is not needed. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
```

Comment:
> see "Cluster capabilities" in _Post-installation configuration_.

1) can we make this an actual clickable link?

2) i dunno that we can fix it in this PR, but i find it a bit odd that we put the capabilities documentation in "post installation configuration" when you need to make your decision about which caps you want to disable *before* you install your cluster.  Yes you can enable caps post-install, but if you don't do anything pre-install, all the caps will be enabled and you won't be able to disable them post-install.  So you really need to know about this/make some decisions prior to installation.

Reply:
Re: your second point
:100: That confused me as well. 

Looping in content strategists for visibility. :wave: 

@sjstout and @mwerner2113: Let's discuss this in #tmp-osdocs-cluster-capabilities next week.